.github/workflows/integration-test-flex-suite.yaml

```diff
@@ -26,30 +26,19 @@ jobs:
         start args: --memory 6g --cpus=2
         github token: ${{ secrets.GITHUB_TOKEN }}
 
-    - name: check k8s cluster status
-      run: |
-        kubectl cluster-info
-        kubectl get pods -n kube-system
+    - name: print k8s cluster status
+      run: tests/scripts/github-action-helper.sh print_k8s_cluster_status
 
     - name: use local disk
       run: |
-        sudo swapoff --all --verbose
-        sudo umount /mnt
-        # search for the device since it keeps changing between sda and sdb
-        PARTITION="/dev/$(lsblk|awk '/14G/ {print $1}'| head -1)1"
-        sudo wipefs --all --force "$PARTITION"
-        sudo lsblk
+        tests/scripts/github-action-helper.sh use_local_disk
```

Comment:
You can remove the `|` and the newline and directly call like:

`run: tests/scripts/github-action-helper.sh use_local_disk`